support-frontend/assets/pages/supporter-plus-landing/components/stickyCta.tsx

```diff
@@ -0,0 +1,65 @@
+import { css, ThemeProvider } from '@emotion/react';
+import {
+	brand,
+	from,
+	space,
+	visuallyHidden,
+} from '@guardian/source-foundations';
+import type { LinkButtonProps } from '@guardian/source-react-components';
+import {
+	buttonThemeReaderRevenueBrand,
+	LinkButton,
+} from '@guardian/source-react-components';
+
+const stickyContainerCss = (isVisible: boolean) => css`
+	display: flex;
+	flex-direction: column;
+	position: sticky;
+	bottom: -4px;
+	background-color: ${brand[400]};
+	padding: ${space[5]}px 0 30px;
+
+	${isVisible ? '' : visuallyHidden}
+
+	${from.tablet} {
+		display: none;
+	}
+`;
+
+const buttonOverrides = css`
+	justify-content: center;
+`;
+
+type StickyCtaProps = {
+	isVisible: boolean;
+	ctaLink: string;
+	onCtaClick: LinkButtonProps['onClick'];
+	buttonText: string;
+	ariaControls: string;
+};
+
+export default function StickyCta({
+	isVisible,
+	ctaLink,
+	onCtaClick,
+	buttonText,
+	ariaControls,
+}: StickyCtaProps): JSX.Element {
+	return (
+		<section css={stickyContainerCss(isVisible)}>
+			<ThemeProvider theme={buttonThemeReaderRevenueBrand}>
+				<LinkButton
+					cssOverrides={buttonOverrides}
+					size="small"
+					onClick={onCtaClick}
+					href={ctaLink}
+					role="button"
+					aria-controls={ariaControls}
```

Comment:
Presume aria? controls and roles for screen readers?

Reply:
[aria-controls](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls) and [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) help to give information to a screenreader about the fact that a particular element manages the visibility of something else on the page.

Overriding the role is _generally_ something you want to avoid, but in this case it's useful to have a link that can navigate the user to the ID of the expanded element, but that can also indicate its expanded/collapsed state to a screenreader (which seems to be behaviour reserved for buttons). So this indicates to assistive technologies that this link should be treated as a button.